docs/source/aws-backend.rst

```diff
@@ -0,0 +1,88 @@
+AWS Backend
+==============
+
+The AWS backend is used to spin up clusters using EC2 instances on Amazon Web Services, where each instance is a DC/OS node.
+
+.. include:: aws-backend-requirements.rst
+
+DC/OS Installation
+------------------
+
+``Cluster``\ s created by the AWS backend only support installing DC/OS via ``install_dcos_from_url``.
```

Comment:
Let's use ``` :py:class:`~dcos_e2e.cluster.Cluster` ``` here and elsewhere in this document, and similar for other linkable items.